gammapy/estimators/points/tests/test_core.py

```diff
@@ -336,3 +337,22 @@ def test_flux_points_plot_no_error_bar():
     flux_points = FluxPoints.from_table(table)
     with mpl_plot_check():
         _ = flux_points.plot(sed_type="dnde")
+
+
+def test_fp_no_is_ul():
+    dataset_1 = simulate_map_dataset(random_state=0, name="dataset_1")
+
+    energy_edges = [0.1, 1, 10, 100] * u.TeV
+
+    fpe = FluxPointsEstimator(
+        energy_edges=energy_edges,
+        norm_n_values=3,
+        source="source",
+    )
+
+    FluxPoints = fpe.run(datasets=dataset_1)
+    table = FluxPoints.to_table()
+
+    if not FluxPoints.has_ul:
+        with pytest.raises(AttributeError):
+            table.is_ul
```

Comment:
```suggestion
    with pytest.raises(ValueError):
        "is_ul" in table.colnames
```

Reply:
Correct me if I wrong but, `"is_ul" in table.colnames` should return False, therefore it will not be a ValueError

Reply:
Yes indeed. But `table.is_ul` will always raise an `AttributeError` since columns are not accessible as attributes on `Table`.
So the correct thing to do is:
```
assert "is_ul" in table.colnames.
```